server/routerlicious/packages/lambdas/src/foreman/lambda.ts

```diff
@@ -99,14 +100,13 @@ export class ForemanLambda implements core.IPartitionLambda {
                         type: "tasks:start",
                     },
                 );
-                this.context.log?.info(
-                    `Request to ${queueName}: ${clientId}:${JSON.stringify(tasks)}`,
-                    {
-                        messageMetaData: {
-                            documentId: docId,
-                            tenantId,
-                        },
-                    });
+                Lumberjack.log(`Request to ${queueName}: ${clientId}:${JSON.stringify(tasks)}`, LogLevel.Info,
+                {
+                    messageMetaData: {
```

Comment:
For Lumberjack, we don't need a top-level "messageMetaData" property in the properties object. Can you instead do something like the below?

```typescript
Lumberjack.info(`Request to ${queueName}: ${clientId}:${JSON.stringify(tasks)}`,
                {
                    [BaseTelemetryProperties.tenantId]: tenantId,
                    [BaseTelemetryProperties.documentId]: documentId,
                }
```

For that, you will also have to import `BaseTelemetryProperties` from  `@fluidframework/server-services-telemetry`